src/OutputWriters/windowed_time_average.jl

```diff
@@ -266,22 +310,25 @@ function advance_time_average!(wta::WindowedTimeAverage, model)
         wta.window_start_time = model.clock.time
         wta.window_start_iteration = model.clock.iteration
         wta.previous_collection_time = model.clock.time
+        # print("elseif end_of_window has been triggered! Setting collecting = false")
 
     elseif end_of_window(wta.schedule, model.clock)
```

Comment:
I don't understand why this is an `elseif` statement. 

Relatedly, I don't understand this comment about not accumulating during the initialization:

https://github.com/CliMA/Oceananigans.jl/blob/56dbae31bb54c1293e12b6ccc88f77976cc1103f/src/OutputWriters/windowed_time_average.jl#L293-L295

Shouldn't this just be
```
if collecting
  if end_of_window
    accumulate_result!(wta, model)
    wta.schedule.collecting = false
  elseif mod(model.clock.iteration - wta.window_start_iteration, stride(wta)) == 0
    accumulate_result!(wta, model)
  end
end
```

Reply:
Afer testing, separating the elseif statements would create NaNs
```julia
 elseif end_of_window(wta.schedule, model.clock)
        # Output is imminent. Finalize averages and cease data collection.
        # Note that this may induce data collecting more frequently than proscribed
        # by `wta.schedule.stride`.
        accumulate_result!(wta, model)
        
        # Averaging period is complete.
        wta.schedule.collecting = false

        # Reset the "previous" interval time, subtracting a sliver that presents overshoot from accumulating.
        initialize_schedule!(wta.schedule, model.clock)
        wta.schedule.actuations += 1

    elseif mod(model.clock.iteration - wta.window_start_iteration, stride(wta)) == 0
        # Collect data as usual
        accumulate_result!(wta, model)
        # wta.window_start_time = model.clock.time - stride(wta) * model.clock.last_Δt
    end
```